### DISCUSSION TITLE
automatic ttyS0/S1 howto ?

### DISCUSSION THREAD

**Discussion by alfonsius** [davmac314/dinit#165] [Q&A] [answered]
please help me with creating config for serial console with login on ttyS1,
i cant find much usefull informatio about inittab-line etc
regards
alf

**Comment by mobin-2008** [comment, marked as answer]
Hi, from `dinit-service(5)` man-page:
> [inittab-id = id-string](https://github.com/davmac314/dinit/wiki/DINIT-SERVICE(5)-man-page#inittab)
    When this service is started, if this setting (or the inittab-line setting) has a specified value, an entry will be created in the system "utmp" database which tracks processes and logged-in users. Typically this database is used by the "who" command to list logged-in users. The entry will be cleared when the service terminates.
>
  > The inittab-id setting specifies the "inittab id" to be written in the entry for the process. The value is normally quite meaningless. However, it should be distinct (or unset) for separate processes. It is typically limited to a very short length.
>
  > The "utmp" database is mostly a historical artifact. Access to it on some systems is prone to denial-of-service by unprivileged users. It is therefore recommended that this setting not be used. However, "who" and similar utilities may not work correctly without this setting (or inittab-line) enabled appropriately.
>
  > This setting has no effect if Dinit was not built with support for writing to the "utmp" database.
[inittab-line = tty-name-string](https://github.com/davmac314/dinit/wiki/DINIT-SERVICE(5)-man-page#inittab~2)
  > This specifies the tty line that will be written to the "utmp" database when this service is started. Normally, for a terminal login service, it would match the terminal device name on which the login process runs, without the "/dev/" prefix.
>
  > See the description of the inittab-id setting for details.

For example:
```
type            = process
command         = /usr/bin/agetty -L -8 ttyS0 115200 vt100
restart         = true
depends-on      = loginready
termsignal      = HUP
smooth-recovery = true
inittab-id      = S0
inittab-line    = ttyS0
```
Regards

**Comment by alfonsius** [comment]
thank you for your fast response, worked at first try :)
